src/typings/mock-fs/mock-fs.d.ts

```diff
@@ -0,0 +1,57 @@
+// Type definitions for mock-fs 3.6.0
+// Project: https://github.com/tschaub/mock-fs
+// Definitions by: Wim Looman <https://github.com/Nemo157>, Qubo <https://github.com/tkqubo>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+
+declare module "mock-fs" {
```

Comment:
It's not. It's used on the run-android tests to mock the file system. I added it in this PR so the .d.ts file won't clutter the other PR.